keystone_api/apps/allocations/factories.py

```diff
@@ -0,0 +1,126 @@
+"""Factories for the allocations app, used for generating test data.
+These factories create instances of models related to allocation requests, allocations, reviews, attachments, comments, and job statistics.
+"""
+
+import random
+from datetime import timedelta
+from django.utils import timezone
+import factory
+from factory import fuzzy
+from factory.django import DjangoModelFactory
+
+from apps.allocations.models import (
+    Allocation,
+    AllocationRequest,
+    AllocationReview,
+    Attachment,
+    Cluster,
+    Comment,
+    JobStats,
+)
+from apps.research_products.models import Grant, Publication
+
+from apps.users.factories import UserFactory, TeamFactory
+from apps.research_products.factories import GrantFactory, PublicationFactory
+
+class ClusterFactory(DjangoModelFactory):
+    """Factory for creating test instances of a Cluster model."""
```

Comment:
Wrap class names in back ticks so it renders as a `code block` when picked up by mkdocs.